src/pages/MyProfile.jsx

```diff
@@ -37,11 +63,11 @@ const MyProfile = () => {
     }, []);
 
     const invalidatePatient = async () => {
-        await api.delete('/patient/' + profileData.id, {
+        await api.delete('/patient', {
             headers: {
                 'Authorization': 'Bearer ' + localStorage.getItem('token')
             }
-        }).catch(() => toast.error("Dogodila se greška."));
+        }).then(navigate('/')).catch(() => toast.error("Dogodila se greška."));

```

Comment:
Add cleaning of context and localStorage. If user is invalidated on Client side there should be no tokens etc.